src/_utils/mediaSizeProvider/MediaSizeProvider.unit.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import MediaSizeProvider, { MediaSizeContext } from './MediaSizeProvider'
+
+const mockWindowInnerWidth = value => {
+  Object.defineProperty(window, 'innerWidth', {
+    writable: true,
+    configurable: true,
+    value,
+  })
+}
+
+describe('MediaSizeProvider', () => {
+  it('Should provide "small" value when screen is under `responsiveBreakpoints.small`', () => {
+    mockWindowInnerWidth(300)
+    const wrapper = shallow(
+      <MediaSizeProvider>
+        <div />
+      </MediaSizeProvider>,
+    )
+
+    expect(wrapper.find(MediaSizeContext.Provider).prop('value')).toEqual('small')
+  })
+
+  it('Should provide "large" value when screen is above `responsiveBreakpoints.small`', () => {
```

Comment:
Don't you want to test that the child component gets the value?

Reply:
Yes I updated the test just now